### PR TITLE
test: my name appears on the page

### LINKED ISSUE #11: Add Eddie's name to the front page (with Cypress test!)

### PR DESCRIPTION
**Pull request by eddieantonio**
I added my name to the front page.

Fixes #11

### CODE REVIEW

File: src/app/App.svelte [hunk 1]
```diff
@@ -4,4 +4,7 @@
 
 <main role="application">
   <h1> Welcome to {name}!</h1>
+
+
```
Comment (IanSeng) [inline src/app/App.svelte]: Suggestion: Remove extra spaces here

**Review by IanSeng — CHANGES_REQUESTED** [review]

**Review by xqiu1 — APPROVED** [review]
LGTM

File: src/app/App.svelte (hunk 1 above)
Comment (IanSeng) [inline src/app/App.svelte]:
```suggestion
```

File: src/app/App.svelte [hunk 2]
```diff
@@ -4,4 +4,7 @@
...
+  <p> Hi my name is Eddie! </p>
```
Comment (IanSeng) [inline src/app/App.svelte:8]:
```suggestion
  <p> Hi my name is Ian! </p>
```

**Review by IanSeng — CHANGES_REQUESTED** [review]

**Review by IanSeng — APPROVED** [review]
👍

**Pull request merged**